libcloud/compute/drivers/gce.py

```diff
@@ -794,9 +794,9 @@ class GCENodeDriver(NodeDriver):
         "PROVISIONING": NodeState.PENDING,
         "STAGING": NodeState.PENDING,
         "RUNNING": NodeState.RUNNING,
-        "STOPPED": NodeState.TERMINATED,
-        "STOPPING": NodeState.TERMINATED,
-        "TERMINATED": NodeState.TERMINATED
+        "STOPPING": NodeState.PENDING,
+        "TERMINATED": NodeState.STOPPED,
```

Comment:
LGTM, please just add a comment here explaining why we are mapping terminated to pending (e.g. gce has no stopped status and node being stopped is represented by terminated status).